src/test/java/com/amazon/ion/streaming/SeekableReaderTest.java

```diff
@@ -1,5 +1,5 @@
-//·Copyright·Amazon.com,·Inc.·or·its·affiliates.·All·Rights·Reserved.
-//·SPDX-License-Identifier:·Apache-2.0
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
This is weird. Do we just have the wrong type of whitespace characters? (Just an observation. No action necessary.)

Reply:
I saw that, but I have no idea how the weird characters made it in there in the first place, given that that header was added by `spotless`.